### PR TITLE
Rearranging content for GS tutorials

### PR DESCRIPTION
**Pull request by runleonarun**
## Description & motivation
 @coap and @runleonarun wanted to address a few concerns with these guides before shipping on April 29th (changed from April 15th to also convert some of the videos into text content.)
* make sure these tutorials are consistent throughout to help people get started in a way that's easy to navigate. 
* We will also be leveraging the new content reuse snippets to reuse code instead of duplicating it.

## To-do before merging

I added priorities so we can get to an MVP.

### General:
- [x] P1 Remove any images that are not necessary to reduce overhead for the page and the reader
- [x] P1 Make “Set up and Connect {{warehouse}}” sections consistent using the following TOC:  
    - Intro
      - Prerequisites (include optional things to check out first (courses.getdbt.com))
      - Setting up
      - Loading data
      - Connecting to dbt Cloud
          * Create dbt Cloud account
           * Enter credentials
      - Next steps (point to next section of the tutorial)
- [x] P1 Fill in the empty sections above
- [x] P2 Make sure we are reusing content that we can
- [x] Add value prop and an intro to Get started (get-started.md )
- [x] P1 Add content to introduction paragraphs in set up guides. 
- [x] P1 Revise get-set-up.md by editing original content and removing the video and possible SQL code.
   - [x] Review these pages (deleted from guide) to determine what needs to become reusables and placed into “Set up and Connect {{warehouse}}” (unless they’re there already)
           "tutorial/create-a-project-dbt-cloud",
- [x] P2 Get as much video content broken up into written steps + screenshots as we can
- [x] P1 Make sure the tutorial gets someone through the 4 activation steps on dbt Cloud
- [x] P1 Make sure warehouse-specific content contains content for all 4 warehouses we're launching with.
- [x] Retitle "Deploy your dbt Project" to "Schedule a job"

### Leveling up section
- [x] P2 Add tabbed code snippets to building your first models and beyond (for each warhouse)
- [x] P3 Edit these  "Build your first models", "Test and Document", "Schedule a job"
- [x] Move dbt CLI content to "Leveling up"  "tutorial/create-a-project-dbt-cli"
   - [x] P3 Maybe consider calling this,  "deploying dbt on your own server" 
- [ ] Add context to the leveling-up.md and building-your-first-project.md landing pages.
  - [ ] Leveling up content can include links to guides  and foundations courses (e.g. we have a module on Jinja and a jinja reference already),
- [x] any other helpful links in connecting to dbt Core for scheduling a job? 

### Release steps
- [x] add any redirects for deleted files.
- [x] Search for any TODO items and resolve.
- [ ] Check and fix any broken links
- [x] Add or remove pages from `website/sidebars.js`
- [ ] Merge into feat/wh-specific-tutorials before merging [this PR](https://github.com/dbt-labs/docs.getdbt.com/pull/1062) into "current"

### CODE REVIEW

**Comment by glin7** [comment]
@runleonarun No preference on my part between those two dates - thanks for the heads-up.

**Comment by runleonarun** [comment]
> @runleonarun No preference on my part between those two dates - thanks for the heads-up.

OK @glin7 let's do US tax day, Monday the 18th. 💰

**Comment by runleonarun** [comment]
@glin7 and @coapacetic, I updated the ship date to April 29th so we can capture some great suggestions from @annafil and Stephanie Yanaga.

**Comment by coapacetic** [comment]
very cool! can you loop in/link the suggestions from Anna and Stephanie when you have the chance?

**Comment by coapacetic** [comment]
FWIW I _believe_ the video content is a supplement to what is already in text so I don't think we will have to do too much video >> text conversion.  Pat created the videos after writing up the entire tutorial in text.  The intention here was to mirror the existing GS tutorial that we are adjusting and because the current BQ videos are outdated. 

This was before we had consensus on where this would live though (docs/devhub/etc.)  Something for us to be mindful of when in the content creation phase based on where things are going to live.

**Comment by runleonarun** [comment]
> can you loop in/link the suggestions from Anna and Stephanie when you have the chance?

@coapacetic I had added these to the OP but looks like I lost them. I will update and add the changes back.

**Review by amychen1776 — COMMENTED** [review]
@runleonarun I think you might have pulled in more things that you might want on this PR; I'm seeing file changes for airflow, sources, etc?  It makes it a bit hard to review

**Comment by runleonarun** [comment]
Sorry about the noise, @amychen1776 the branch just fell way behind `feat/wh-specific-tutorials`. I just updated so it should be good now.

File: website/docs/faqs/example-projects.md [hunk 1]
```diff
@@ -3,7 +3,8 @@ title: Are there any example dbt projects?
 ---
 
 Yes!
-* **Getting Started Tutorial:** You can build your own example dbt project in the [Getting Started Tutorial](tutorial/1-setting-up.md)
+
+* **Getting Started Tutorial:** You can build your own example dbt project in the [Getting Started Tutorial](tutorial/getting-started.md)
```
Comment (amychen1776) [inline website/docs/faqs/example-projects.md:7]: I think it might be helpful to add in how long this could take for context

File: website/docs/learn/1-setting-up.md [hunk 2]
```diff
@@ -12,7 +12,7 @@ If you are **new to dbt**, we recommend you take some time to read through the F
 If you are already **experienced with dbt**, you still need to complete this tutorial as we'll be building on this project in our lessons.
 
 :::info
-Please note that this tutorial is adapted from the main <a href="https://tutorial.getdbt.com/tutorial/setting-up/">Getting Started</a> tutorial — there may be some references to BigQuery in various videos, however the same concepts should apply on Snowflake.
+Please note that this tutorial is adapted from the main <a href="https://tutorial.getdbt.com/tutorial/getting-started">Getting Started</a> tutorial — there may be some references to BigQuery in various videos, however the same concepts should apply on Snowflake.
```
Comment (amychen1776) [inline website/docs/learn/1-setting-up.md]: Should we be updating this to say concepts apply across data platforms? To be more inclusive of data warehouses outside of BQ and Snowflake

File: website/docs/tutorial/1a-setting-up-bigquery.md [hunk 3]
```diff
@@ -1,11 +1,19 @@
 ---
-title: "Setting up BigQuery for dbt Cloud"
+title: "Set up and connect BigQuery"
 id: setting-up-bigquery
-description: "Set up BigQuery with sample data and connect to dbt Cloud."
-sidebar_label: "Setting up with BigQuery"
+description: "Set up BigQuery with sample data and connect to dbt Cloud or dbt CLI."
```
Comment (amychen1776) [inline website/docs/tutorial/1a-setting-up-bigquery.md]: I know folks call it dbt CLI but our product is dbt Core

File: website/docs/tutorial/1a-setting-up-bigquery.md [hunk 4]
```diff
@@ -1,11 +1,19 @@
...
 ---
 
-## Create a BigQuery Project
+## Introduction
+
+Add intro
+## Prerequisites
+
+* Existing Cloud Provider account (AWS, GCP, Azure).
+* Permissions to create S3 bucket in you cloud provider account.
```
Comment (amychen1776) [inline website/docs/tutorial/1a-setting-up-bigquery.md]:
```suggestion
* Permissions to create S3 bucket in your cloud provider account.
```

File: website/docs/tutorial/1a-setting-up-bigquery.md [hunk 5]
```diff
@@ -225,3 +214,9 @@ Above we created a project in Bigquery and created an API key so we interact wit
         <Lightbox src="/img/bigquery/preview-data.png" title="dbt Cloud - Statement Preview Project Data" />
 
 Congratulations, you are all set! 
+
+### Create dbt Cloud account
```
Comment (amychen1776) [inline website/docs/tutorial/1a-setting-up-bigquery.md]: Are we missing things here?

File: website/docs/tutorial/1b-setting-up-snowflake.md [hunk 6]
```diff
@@ -1,11 +1,16 @@
...
-description: "Set up Snowflake with sample data and connect to dbt Cloud."
-sidebar_label: "Setting up with Snowflake"
+description: "Set up Snowflake with sample data and connect to dbt Cloud or dbt Core."
+sidebar_label: "Set up and connect Snowflake"
 ---
 
-## Get started with Snowflake
+## Intro
+
+## Prerequisites
```
Comment (amychen1776) [inline website/docs/tutorial/1b-setting-up-snowflake.md]: Missing things here?

File: website/docs/tutorial/1b-setting-up-snowflake.md [hunk 7]
```diff
@@ -309,4 +314,11 @@ Awesome!  You are all set to leverage the power of dbt Cloud and Snowflake toget
...
 * Connected dbt Cloud and Snowflake
-* Set up the dbt Cloud IDE, queried data, and did your first dbt run
\ No newline at end of file
+* Set up the dbt Cloud IDE, queried data, and did your first dbt run
+
+### Create dbt Cloud account
+
+### Enter connection credentials
+
+## Next steps
```
Comment (amychen1776) [inline website/docs/tutorial/1b-setting-up-snowflake.md]: Maybe I'm missing the use of the headers without text underneath?

File: website/docs/tutorial/getting-set-up.md [hunk 8]
```diff
@@ -0,0 +1,72 @@
+---
+title: Getting set up
+id: getting-set-up
+description: "Set up your environment so you can start using dbt."
+---
+
+Learn how to turn the following SQL query into a tested, documented, and deployed dbt project. You can preview [the generated documentation](https://www.getdbt.com/getting-started-tutorial/#!/overview) for the project you're about to build. You'll need a working knowledge of SQL to complete this tutorial.
```
Comment (amychen1776) [inline website/docs/tutorial/getting-set-up.md]: I think we want to give an time estimation here. I usually say our getting started takes about 1 hour to do but I don't know if @coapacetic would agree with me.

File: website/docs/tutorial/getting-set-up.md [hunk 9]
```diff
@@ -0,0 +1,72 @@
...
+
+If you want a more in-depth learning experience, consider taking some [dbt Learn online courses](https://courses.getdbt.com/).
```
Comment (amychen1776) [inline website/docs/tutorial/getting-set-up.md]:
```suggestion
If you want a more in-depth learning experience, we recommend taking the dbt Fundamentals on our [dbt Learn online courses site](https://courses.getdbt.com/).
```

File: website/docs/tutorial/leveling-up.md [hunk 10]
```diff
@@ -0,0 +1,6 @@
+---
+title: "Leveling up"
```
Comment (amychen1776) [inline website/docs/tutorial/leveling-up.md]: Not sure what Leveling is up for but I want to plug for the courses if it means building on what they learned in the getting started guide?

**Review by amychen1776 — COMMENTED** [review]
Thanks for fixing the commits! I took the first pass at it and it looks great. Left some suggestions and comments; I think I might be misunderstanding how you're reusing the code so please ignore it if it's not relevant.

**Comment by coapacetic** [comment]
I had a fun time making some contributions here this morning.  Here is a quick summary for those following along and some additional open questions

Contributions:
- I did a full rewrite of the BQ and SF tutorials with the proposed ToC.  See open questions below for some follow ups.  These definitely need another read and trial to make sure they good to go.  I removed images that I felt weren't required. In cases where I wasn't sure, I defaulted to leaving the images.
- I created two reusable snippets that are used in both the BQ and SF tutorials.  These can be found under `website/docs/snippets` and are prefixed with the word `tutorial`

Open questions:
- BigQuery tutorial: The current tutorial explains how to create views in your BQ project on top of the public datasets.  Should we keep this or shorten this to just being querying the public datasets like the original tutorial and dbt fundamentals?
- When instructing the reader to click a button, do we like the word "click" over "select"?  Small style thing I made a note of.
- On my local machine, I tried to make a component for displaying code snippets for all four warehouse really easily. This is beyond my frontend knowledge, so would love someone to take a look. I have this currently mapped out _without a component_ in `tutorial-managed-repo-and-initiate-project`.  My vision is that this would act like the `CloudCore` component.
- With the introduction of SF and DB partner connect, we might revisit the ToC slightly to make it consistent but adaptable to this option for setting up dbt Cloud.  This is hard to propose her in a PR, so happy to hop on a huddle to screenshare what I'm thinking.

**Comment by coapacetic** [comment]
> how you're reusing the code so please ignore it if it's not relevant. 

@amychen1776 I just tested this out with the latest commits.  Check out the following to see how I implemented this if you are curious:
- `/website/snippets/tutorial-managed-repo-and-initiate-project.md` is the reusable snippet
- this is referenced in `website/docs/tutorial` in both the BQ and SF tutorials at the moment.

**Comment by runleonarun** [comment]
> I created two reusable snippets that are used in both the BQ and SF tutorials. These can be found under website/docs/snippets and are prefixed with the word tutorial

😍 

> When instructing the reader to click a button, do we like the word "click" over "select"? Small style thing I made a note of.

My preference is to click "a thing" but select "a thing" from a list. For example, "click **Big**" not "Click the Big button" or when using select: Click **Menu** and select **Big**. 

> Biq Query tutorial: 

I'm leaning towards querying the public datasets and keeping it simple. We should chat about this and what the tradeoffs are in case I'm not fully grasping that.  

> On my local machine, I tried to make a component for displaying code snippets for all four warehouse really easily. This is beyond my frontend knowledge, so would love someone to take a look. I have this currently mapped out without a component in tutorial-managed-repo-and-initiate-project. My vision is that this would act like the CloudCore component.
  With the introduction of SF and DB partner connect, we might revisit the ToC slightly to make it consistent but adaptable to this option for setting up dbt Cloud. This is hard to propose her in a PR, so happy to hop on a huddle to screenshare what I'm thinking.

For these two, let's have a quick sync chat so I can make sure I understand. I'm 100% good with modifying the toc to make it clearer for readers.

File: website/snippets/tutorial-next-steps-setting-up.md [hunk 11]
```diff
@@ -0,0 +1 @@
+You can **[Continue the tutorialby building your first project](/building-your-first-project)**. You will learn and apply the fundamentals of dbt with models, tests, docs, and deploying your first job!
```
Comment (runleonarun) [inline website/snippets/tutorial-next-steps-setting-up.md]:
```suggestion
You can **[Continue the tutorial by building your first project](/building-your-first-project)**. You will learn and apply the fundamentals of dbt with models, tests, docs, and deploying your first job!
```

**Comment by coapacetic** [comment]
Bubbling up some takeaways from an awesome sync that Leona and I had:

**Table of Contents**
- For ToC, we will have the following top level headers: Prerequisites, Setting up, Loading data, Connecting to dbt Cloud, Next steps.
- The subheadings for Connecting to dbt Cloud will vary by warehouse to accommodate differences in warehouses and support for partner connected on Snowflake and Databricks.

**WHCore Code snippets**
- **Initial solution:** In the warehouse set up guides, we have successfully built our tabbed code snippets using <Tabs> in a reusable snippet.

image

- This is likely a common structure we want to use for supporting multiple warehouses in a single snippet throughout docs.
- I've attempted to create a component similar to `CloudCore` that will support this. I've called it `WHCode`.
- I'm able to get this working for plain text, but it is not supporting nicely formatted code snippets like we are seeing in the initial solution above.  All of my work on this was in the `warehouse tabs as component` commit above.  

image

- @JKarlavige if you have the time, can you take a look at this for a potential work around. This is beyond my hacky understanding of React/JS 😅

File: website/docs/tutorial/1b-setting-up-snowflake.md [hunk 12]
```diff
@@ -206,107 +229,68 @@ There are two ways to connect dbt Cloud and Snowflake. The first option is Partn
...
+#### This code snippet is done using the WHCode component
 
-12. Congrats! Your account is officially set up and ready to rock. 
+<WHCode>
 
-## Start developing in dbt Cloud
+<p><code>I found some code</code></p>
+<p><code class="language-sql">select * from customers</code></p>
+<p>```I was hoping this code would be nicely formatted```</p>
+<p>Some text!</p>
```
Comment (JKarlavige) [inline website/docs/tutorial/1b-setting-up-snowflake.md]: Hey @coapacetic, it looks like markdown code blocks require a line break above and below the snippet. Updating that line to have a space before and after should allow it to be formatted properly.

```suggestion
<p><code>I found some code</code></p>
<p><code class="language-sql">select * from customers</code></p>

<p>```I was hoping this code would be nicely formatted```</p>

<p>Some text!</p>
```

Comment (JKarlavige) [reply website/docs/tutorial/1b-setting-up-snowflake.md]: image

File: website/docs/tutorial/1b-setting-up-snowflake.md (hunk 7 above)
Comment (runleonarun) [reply website/docs/tutorial/1b-setting-up-snowflake.md]: Content was just not ready yet!

File: website/docs/tutorial/1a-setting-up-bigquery.md (hunk 3 above)
Comment (runleonarun) [reply website/docs/tutorial/1a-setting-up-bigquery.md]: Great callout. I will edit for this.

File: website/docs/tutorial/1b-setting-up-snowflake.md (hunk 6 above)
Comment (runleonarun) [reply website/docs/tutorial/1b-setting-up-snowflake.md]: Yep. We are still filling in content.

File: website/docs/tutorial/1b-setting-up-snowflake.md (hunk 12 above)
Comment (coapacetic) [reply website/docs/tutorial/1b-setting-up-snowflake.md]: thanks Jason!!!  I was able to hammer away at this a little bit more with Leona this afternoon and get things working with some whitespace and div's.  Moving forward, the new component is `WHCode` for tabbed warehouse specific code snippets.  When we add additional warehouses, we can extend this to 5, 6, 7... tabs

**Comment by coapacetic** [comment]
Just completed "Make “Set up and Connect {{warehouse}}” sections consistent using the following TOC:"  Each of the warehouse specific sections still need a few more edits for a consistent presentation between each, but the TOC structure is there. We are getting there 🏃 🎉

File: _redirects [hunk 13]
```diff
@@ -289,3 +289,8 @@ https://tutorial.getdbt.com/*	https://docs.getdbt.com/:splat	301!
 /reference/project-configs/modules-paths  /reference/project-configs/packages-install-path 302
 /docs/dbt-cloud/using-dbt-cloud/cloud-slack-notifications   /docs/dbt-cloud/using-dbt-cloud/cloud-notifications 302
 /reference/warehouse-profiles/presto-profile    /reference/profiles.yml 302
+/tutorial/setting-up	/tutorial/getting-started	302
+/tutorial/test-and-document-your-project	/tutorial/building-your-first-project/test-and-document-your-project	302
+/tutorial/build-your-first-models	/tutorial/building-your-first-project/build-your-first-models	302
+/tutorial/deploy-your-project /tutorial/building-your-first-project/deploy-your-project	302
+/tutorial/using-jinja	/tutorial/learning-more/using-jinja	302
```
Comment (krevitt) [inline _redirects:296]: @runleonarun noticing that we may be missing a redirect for the refactoring legacy SQL tutorial (from /tutorial/refactoring-legacy-sql to /tutorial/learning-more/refactoring-legacy-sql)

Comment (runleonarun) [reply _redirects:296]: Good catch @krevitt I will add that to the follow-up issue and start a PR with that chnage.
